packages/@aws-cdk/aws-cloudfront/lib/cache-policy.ts

```diff
@@ -0,0 +1,305 @@
+   * @default - The greater of 1 year and ``defaultTtl``
+   */
+  readonly maxTtl?: Duration;
+
+  /**
+   * Determines whether any cookies in viewer requests are included in the cache key and automatically included in requests that CloudFront sends to the origin.
+   * @default CookieBehavior.none()
+   */
+  readonly cookieBehavior?: CacheCookieBehavior;
+
+  /**
+   * Determines whether any HTTP headers are included in the cache key and automatically included in requests that CloudFront sends to the origin.
+   * @default HeaderBehavior.none()
+   */
+  readonly headerBehavior?: CacheHeaderBehavior;
+
+  /**
+   * Determines whether any query strings are included in the cache key and automatically included in requests that CloudFront sends to the origin.
+   * @default QueryStringBehavior.none()
+   */
+  readonly queryStringBehavior?: CacheQueryStringBehavior;
+
+  /**
+   * Whether to normalize and include the `Accept-Encoding` header in the cache key whem the `Accept-Encoding` header is 'gzip'.
+   * @default false
+   */
+  readonly enableAcceptEncodingGzip?: boolean;
+}
+
+/**
+ * A Cache Policy configuration.
+ *
+ * @resource AWS::CloudFront::CachePolicy
+ * @experimental
+ */
+export class CachePolicy extends Resource implements ICachePolicy {
+
+  /**
+   * Optimize cache efficiency by minimizing the values that CloudFront includes in the cache key.
+   * Query strings and cookies are not included in the cache key, and only the normalized 'Accept-Encoding' header is included.
+   */
+  public static readonly CACHING_OPTIMIZED = CachePolicy.fromManagedCachePolicy('658327ea-f89d-4fab-a63d-7e88639e58f6');
+  /**
+   * Optimize cache efficiency by minimizing the values that CloudFront includes in the cache key.
+   * Query strings and cookies are not included in the cache key, and only the normalized 'Accept-Encoding' header is included.
+   * Disables cache compression.
+   */
+  public static readonly CACHING_OPTIMIZED_FOR_UNCOMPRESSED_OBJECTS = CachePolicy.fromManagedCachePolicy('b2884449-e4de-46a7-ac36-70bc7f1ddd6d');
+  /** Disables caching. This policy is useful for dynamic content and for requests that are not cacheable. */
+  public static readonly CACHING_DISABLED = CachePolicy.fromManagedCachePolicy('4135ea2d-6df8-44a3-9df3-4b5a84be39ad');
+  /** Designed for use with an origin that is an AWS Elemental MediaPackage endpoint. */
+  public static readonly ELEMENTAL_MEDIA_PACKAGE = CachePolicy.fromManagedCachePolicy('08627262-05a9-4f76-9ded-b50ca2e3a84f');
+
+  /** Imports a Cache Policy from its id. */
+  public static fromCachePolicyId(scope: Construct, id: string, cachePolicyId: string): ICachePolicy {
+    return this.fromCachePolicyAttributes(scope, id, { cachePolicyId });
+  }
+
+  /** Imports a Cache Policy from its attributes. */
+  public static fromCachePolicyAttributes(scope: Construct, id: string, attrs: CachePolicyAttributes): ICachePolicy {
```

Comment:
If the only interesting attribute of an imported `CachePolicy` is its ID, just remove this method, and the `CachePolicyAttributes` interface. Just `fromCachePolicyId` is fine.

We can always add `fromCachePolicyAttributes` later if we need it.